<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjetoFinal</class>
 <widget class="QMainWindow" name="ProjetoFinal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProjetoFinal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="logText">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>521</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>681</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="inputImg">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="outputImg">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>390</y>
      <width>87</width>
      <height>77</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="pastButtom">
       <property name="text">
        <string>Past</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="futureButtom">
       <property name="text">
        <string>Future</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="text">
        <string>CheckBox</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>inputImg</zorder>
   <zorder>outputImg</zorder>
   <zorder>logText</zorder>
   <zorder>FutureButtom</zorder>
   <zorder>pastButtom</zorder>
   <zorder>radioButton_3</zorder>
   <zorder>inputImg</zorder>
   <zorder>pastButtom</zorder>
   <zorder>futureButtom</zorder>
   <zorder>checkBox_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>923</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
